pkg/management/component.go

```diff
@@ -1386,6 +1416,50 @@ func (c *component) GetRole(ctx context.Context, realmName string, roleID string
 	return roleRep, nil
 }
 
+func (c *component) CreateRole(ctx context.Context, realmName string, role api.RoleRepresentation) (string, error) {
+	var accessToken = ctx.Value(cs.CtContextAccessToken).(string)
+
+	var roleRep = api.ConvertToKCRole(role)
```

Comment:
Add Business flag here instead of into conversion method

Reply:
Corrected